src/main/java/frc/robot/subsystems/RollerSubsystem.java

```diff
@@ -26,16 +30,34 @@ public RollerSubsystem() {
         rightBeak = MotorUtil.createTalonSRX(RIGHT_ID);
         rightBeak.follow(leftBeak);
         rightBeak.setNeutralMode(NeutralMode.Brake);
+
+        openMotor = MotorUtil.createTalonSRX(OPEN_ID);
+        openMotor.setNeutralMode(NeutralMode.Brake);
+    }
+
+    public void OpenMotor(){
```

Comment:
nitpicky, but methods should be lowerCamelCase, only constructors are UpperCamelCase